### PR TITLE
Enhancement : Introduce failsafe mechanism for jinja-rendering

### PR DESCRIPTION
**Pull request by mithunbharadwaj**

### CODE REVIEW

**Comment by ateska** [comment]
That email is not very readable, honestly.

Can you please insert some newlines etc. so that is it not that "compact"?

**Review by ateska — CHANGES_REQUESTED** [review]

File: asabiris/orchestration/sendemail.py [hunk 1]
```diff
@@ -117,34 +124,66 @@ async def render(self, template, params):
...
+				}
+
+				raw_dump = json.dumps(exception_details, indent=4)
+				jinja_exception_output = "{}\n\nException Details:\n{}".format(error_message, raw_dump)
+				subject = "Jinja2 Rendering Error"
+				return jinja_exception_output, subject
+
+		except Exception as e:
+			# General catch-all for exceptions
+			error_message = "An unexpected error occurred during rendering."
```
[the remark this answers is not in the document]
Comment (mithunbharadwaj) [reply asabiris/orchestration/sendemail.py:172]: @ateska yes, I can do that!

[the remark this answers is not in the document]
Comment (ateska) [reply asabiris/orchestration/sendemail.py:172]: You can either use HTML or send a plain text email ... I suggest the second option.
You need to use `\r\n` as line ends -> https://laravel.io/forum/09-04-2014-plain-text-email-line-breaks

[the remark this answers is not in the document]
Comment (mithunbharadwaj) [reply asabiris/orchestration/sendemail.py:172]: Screenshot 2023-08-25 at 14 25 38

**Review by ateska — APPROVED** [review]

File: asabiris/app.py [hunk 2]
```diff
@@ -32,6 +32,9 @@
 	"web": {
 		"listen": 8896  # Well-known port of asab iris
 	},
+	"smtp": {
```
Comment (ateska) [inline asabiris/app.py:35]: this should be:

```
[jinja]
failsafe=True
```

**Review by ateska — APPROVED** [review]

**Pull request merged**